src/lib/utils/skin.ts

```diff
@@ -165,7 +165,7 @@ export function isBlueSkin(itemInfo: ItemInfo): boolean {
 
 export function getFadeCalculatorAndSupportedWeapon(
```

Comment:
`getFadeCalculatorAndSupportedWeapon` is outdated, we're also returning the `Fade Type` now.

Perhaps `getFadeParams` or similar to make this more generic.